packages/roosterjs-editor-plugins/lib/plugins/TableSelection/TableSelection.ts

```diff
@@ -0,0 +1,195 @@
+import { EditorPlugin, IEditor, Keys, PluginEvent, PluginEventType } from 'roosterjs-editor-types';
+import {
+    Browser,
+    clearSelectedTableCells,
+    highlightTableSelection,
+    setTableSelectedRange,
+} from 'roosterjs-editor-dom';
+
+/**
+ * TableSelectionPlugin help highlight table cells
+ */
+export default class TableSelectionPlugin implements EditorPlugin {
```

Comment:
Consider make this a core plugin, because the table selection is also something editor core should know